src/Monolog/Processors/CommonProcessor.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace Uniplaces\Monolog\Processors;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+/**
+ * CommonProcessor
+ */
+class CommonProcessor
+{
+    /**
+     * @var RequestStack
+     */
+    protected $requestStack;
+
+    /**
+     * @var string
+     */
+    protected $appId;
+
+    /**
+     * @var string
+     */
+    protected $gitHash;
+
+    /**
+     * @var string
+     */
+    protected $env;
+
+    /**
+     * @param RequestStack $requestStack
+     * @param string       $appId
+     * @param string       $gitHash
+     * @param string       $env
+     */
+    public function __construct(RequestStack $requestStack, string $appId, string $gitHash, string $env)
+    {
+        $this->requestStack = $requestStack;
+        $this->appId = $appId;
+        $this->gitHash = $gitHash;
+        $this->env = $env;
+    }
+
+    /**
+     * @param array $record Each record represents one line entry
+     *
+     * @return array
+     */
+    public function processRecord(array $record): array
+    {
+        $record = $this->buildCommon($record);
+        if ($this->hasHttpRequest()) {
+            return $this->buildHttpLog($record, $this->requestStack->getCurrentRequest());
+        }
+
+        return $this->buildCommandLog($record);
+    }
+
+    /**
+     * @param array $record
+     *
+     * @return array
+     */
+    protected function buildCommon(array $record): array
+    {
+        $commonFields = [
+            'time' => time(),
+            'app-id' => $this->appId,
+            'env' => $this->env,
+            'msg' => $record['message'],
+            'level' => strtolower($record['level_name']),
```

Comment:
Test line 46 checks if `level === 0` 
I'm probably missing something, but how can it be `0` if it's a lowercase version of the name?